packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/applyChange.ts

```diff
@@ -73,9 +74,28 @@ export default function applyChange(
     image.width = targetWidth;
     image.height = targetHeight;
 
+    // Keep Image responsive, if not resized
+    setImageResponsive(image, wasResized);
+
     return (
         srcChanged ||
         editInfo.widthPx != initEditInfo.widthPx ||
         editInfo.heightPx != initEditInfo.heightPx
     );
 }
+
+/**
+ * Check if the image must be responsive
+ * @param img The current image.
+ * @param wasResized the current resize state of the image
+ */
+function setImageResponsive(img: HTMLImageElement, wasResized: boolean) {
+    if (wasResized) {
+        img.style.maxWidth = 'initial';
+        img.removeAttribute('height');
```

Comment:
make sure this won't cause any exception, test in all browsers (Chrome, Safari, Firefox, Edge, IE)

Reply:
I tested in all browsers.